src/cli.ts

```diff
@@ -181,6 +182,12 @@ const yarg = yargs(process.argv.slice(2))
           alias: 'i',
           describe: 'Path for the json translations',
         },
+        out: {
+          type: 'string',
+          demandOption: true,
```

Comment:
I think `demandOption: true` makes it mandatory

Reply:
Removed